packages/cms-cli/commands/customObject.js

```diff
@@ -1,15 +1,41 @@
 const { addConfigOptions, addPortalOptions } = require('../lib/commonOpts');
 const schemaCommand = require('./customObject/schema');
 const createCommand = require('./customObject/create');
+const chalk = require('chalk');
 
 exports.command = ['custom-object', 'custom', 'co'];
-// This hides the command from --help until we are ready to release it
-exports.describe = false; // 'Manage Custom Objects';
+exports.describe =
+  'Manage Custom Objects.  This feature is currently in beta and the CLI contract is subject to change';
+
 exports.builder = yargs => {
   addConfigOptions(yargs, true);
   addPortalOptions(yargs, true);
 
-  yargs.command(schemaCommand).command(createCommand);
+  yargs
+    .command(schemaCommand)
+    .command(createCommand)
+    .demandCommand(1, '');
+
+  console.warn(
+    chalk.reset.yellow(
+      '***************************** WARNING WARNING WARNING ****************************'
+    )
+  );
+  console.warn(
+    chalk.reset.yellow(
+      'The Custom Object CLI is currently in beta and is subject to change.'
+    )
+  );
+  console.warn(
+    chalk.reset.yellow(
+      'See https://developers.hubspot.com/docs/api/crm/crm-custom-objects to find out more.'
+    )
+  );
+  console.warn(
+    chalk.reset.yellow(
+      '***************************** WARNING WARNING WARNING ****************************'
+    )
+  );
```

Comment:
Can we use the `logger` for this?

Reply:
So I was going to at first, but it wasn't showing up cuz the default logger level didn't show warning.  I figured we always want this to show

Reply:
I could add a new level to logger, like `logger.announce`?

Reply:
Yeah, I think something like that makes sense. Since this is on the CLI side instead of inside `cms-lib`, it isn't a big deal.

Reply:
👍 I'ma keep it as is for now